src/main/scala/org/broadinstitute/dsde/rawls/workspace/WorkspaceService.scala

```diff
@@ -1161,6 +1164,104 @@ class WorkspaceService(protected val userInfo: UserInfo, dataSource: DataSource,
     }
   }
 
+
```

Comment:
its outside the scope of this because this is workspace status, but we might want to also have validation that each user is in the all users group, and validation endpoints to verify that only members in a given group have access to some bucket that has acls only for that group (which we would use to validate that the TCGA protected bucket can only be accessed by the whitelist group)